### PR TITLE
feat: add functionality for toolbar mention block

### PR DESCRIPTION
**Pull request by evd3v**
##### Summary

Adds a type-ahead / drop-down for looking up local Pod users, feeds and nicknames of feeds a user is following.

##### Component Name

- area/ui

##### Test Plan

Screen Shot 2020-08-09 at 07 27 38
Screen Shot 2020-08-09 at 07 27 44

### CODE REVIEW

File: internal/static/js/twtxt.js [hunk 1]
```diff
@@ -223,6 +240,83 @@ u('#imgBtn').on("click", function(e) {
...
+    u("#mentioned-list").first().classList.remove('show')
+  }
+})
+
+u("textarea#text").on("focus", (e) => {
+  if(e.relatedTarget === u('#usrBtn').first()) {
+    u("#mentioned-list").first().style.top = u("textarea#text").first().clientHeight + 2 + 'px'
+    u("#mentioned-list").first().classList.add('show')
+
+    fetch('/lookup').then((res) => res.json()).then(res => {
```
Comment (prologic) [inline internal/static/js/twtxt.js:257]: Ahh! Let's use the `Twix` lib here (_already included in the project_); Or this won't work on IE :)

Comment (prologic) [reply internal/static/js/twtxt.js:257]: Also don't use `=>` functions either :)

**Comment by dooven** [comment]
Add some screenshots maybe? Would be nice to see the changes without having to run it locally

**Comment by evd3v** [comment]
1) Fix IE bugs (with fetch request - changed to **Twix.ajax**)
2) Add night mode for mention users list
3) Add behavior with press tab after list of mention users is active (after press tab it select the first result)
4) Add behavior with typing @ (same with pressing button in the toolbar)
5) Refactoring some functions

Cant attach video with result, cause it's doesn't support.
Cant show behavior by screenshots

File: internal/static/js/twtxt.js [hunk 2]
```diff
@@ -189,7 +205,38 @@ function insertText(selector, text) {
...
+
+  if(searchStr) {
+    requestUrl += '?prefix=' + searchStr;
+  }
+
+  Twix.ajax({
+    type: "GET",
+    url: requestUrl,
+    success: function(data) {
+      var nodes = data.map((user) => {
```
Comment (prologic) [inline internal/static/js/twtxt.js:226]: IE Bug: Right arrow func.

File: internal/static/js/twtxt.js [hunk 3]
```diff
@@ -223,6 +270,77 @@ u('#imgBtn').on("click", function(e) {
...
+  }
+})
+
+u("textarea#text").on("input", (e) => {
+  var value = u("textarea#text").first().value;
+
+  if($mentionedList.classList.contains('show')) {
+    var lastIndex = getLastMentionIndex(value);
+
+    console.log(lastSymbol)
```
Comment (dooven) [inline internal/static/js/twtxt.js:297]: remove log

**Pull request merged**